### PR TITLE
ui: ILL BorrowingRequest details page

### PR DESCRIPTION
**Pull request by ntarocco**
localhost_3000_backoffice_ill_borrowing-requests_m1m3f-qjp89-2

localhost_3000_backoffice_ill_borrowing-requests_m1m3f-qjp89_edit(iPad Pro)

localhost_3000_backoffice_acquisition_orders_jw2xk-8a157-3

### CODE REVIEW

File: invenio_app_ils/ill/schemas/ill_borrowing_requests/borrowing_request-v1.0.0.json [hunk 1]
```diff
@@ -186,7 +186,7 @@
       "title": "Order total in the main currency"
     },
     "type": {
-      "title": "The type of the item requested, normally physical or electronic",
+      "title": "The type of the item requested, normally physical or electronic (Vocabulary)",
```
Comment (kpsherva) [inline invenio_app_ils/ill/schemas/ill_borrowing_requests/borrowing_request-v1.0.0.json:189]: nice, we should do it everywhere

File: ui/src/api/documents/serializer.js [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
-      result['links'] = hit.links;
-    }
-    if (!isEmpty(hit.metadata)) {
-      result['metadata'] = hit.metadata;
-      if (!isEmpty(hit.metadata.circulation.next_available_date)) {
-        result['metadata']['circulation']['next_available_date'] = new Date(
-          result['metadata']['circulation']['next_available_date']
-        ).toDateString();
+const DocumentSerializers = {
+  DATE_FIELDS: ['circulation.next_available_date'],
```
Comment (kpsherva) [inline ui/src/api/documents/serializer.js:7]: this looks much cleaner :rocket:

File: ui/src/pages/backoffice/Acquisition/Order/OrderDetails/OrderDetails.js [hunk 3]
```diff
@@ -98,6 +100,14 @@ class ActionMenu extends React.Component {
       <div className={'bo-action-menu'}>
         <EditButton fluid to={AcquisitionRoutes.orderEditFor(order.pid)} />
 
+        <Message size="small">
```
Comment (kpsherva) [inline ui/src/pages/backoffice/Acquisition/Order/OrderDetails/OrderDetails.js:93]: it is not bad looking in the end, I think the information is important

**Review by kpsherva — APPROVED** [review]

**Comment by ntarocco** [comment]
@kprzerwa I am not able to fix the menu in the details pages of Orders and Borrowing Requests with this `ref` madness, help needed! :)

File: invenio_app_ils/cli.py [hunk 4]
```diff
@@ -818,18 +825,31 @@ def generate(self):
                 "library_pid": self.random_library_pid(),
                 "document_pid": self.random_document_pid(),
                 "patron_pid": random.choice(self.holder.patrons_pids),
-                "type": "Hardcover",
+                "type": "ELECTRONIC",
```
Comment (kpsherva) [inline invenio_app_ils/cli.py:828]: is the type medium of the ILL Item?

Comment (ntarocco) [reply invenio_app_ils/cli.py:828]: Will change in the next on about ILL ;)

File: invenio_app_ils/ill/schemas/ill_borrowing_requests/borrowing_request-v1.0.0.json (hunk 1 above)
Comment (ntarocco) [reply invenio_app_ils/ill/schemas/ill_borrowing_requests/borrowing_request-v1.0.0.json:189]: It is done extensively in the ACQ and ILL, we can apply it to the rest.

**Pull request merged**